nservicebus/upgrades/7to8/index.md

```diff
@@ -11,6 +11,38 @@ upgradeGuideCoreVersions:
 
 NOTE: This is a working document; there is currently no timeline for the release of NServiceBus version 8.0.
 
+## Transport configuration
+
+NServiceBus V8 comes with a new transport configuration API. Instead of the generic based `UseTransport<TTransport>` method, create an instance of the transport's configuration class and pass it to `UseTransport`, e.g.
+
+```csharp
+var transport = endpointConfiguration.UseTransport<MyTransport>();
+
+transport.Transactions(TransportTransactionMode.ReceiveOnly);
+var routing = t.Routing();
+routing.RouteToEndpoint(typeof(MyMessage), "DestinationEndpoint");
+```
+
+becomes:
+
+```csharp
+var transport = new MyTransport{
+    TransportTransactionMode = TransportTransactionMode.ReceiveOnly
+};
+
+var routing = endpointConfiguration.UseTransport(transport);
+routing.RouteToEndpoint(typeof(MyMessage), "DestinationEndpoint");
+```
+
+The transport can be configured via the transport's configuration type which derives from `TransportDefinition`. See the [transport upgrade guide](/nservicebus/upgrades/7to8/transport.md) for further details.
+
+## Connection strings
```

Comment:
should this section be moved to the transport specific page or do we keep this here?

Reply:
I think so, especially because there are transports for which a connection string doesn't make sense.